src/src/version.py

```diff
@@ -9,3 +9,4 @@
 Visit https://github.com/spdx/tools-java/releases to know about the tools releases.
 """
 java_tools_version = '1.1.3'
+ntia_conformance_checker_version = '0.1.0'
```

Comment:
I believe there should be a way to get this version from `ntia-conformance-checker` package itself when it is released. Something like `from ntia-conformance-checker import version` or `ntia_conformance_checker.version`. We can dig into this later on. Similarly for others.

Reply:
Really good point - I'll create a separate issue to add this in later